common/Mesh-Network-Introduction.md

```diff
@@ -0,0 +1,124 @@
+
+A common strategy in wireless network is to use 2400 MHz which was the one of the first worldwide unlicensed bands and is a very commonly-used frequency band. Both Bluetooth SIG and Zigbee Alliance endorse the use of this band, and of course the Zigbee Alliance can also use other SubGHz bands to expand their capabilities.   
+
+## 1.1. Network Topology
+Before taking about the mesh network, we'd like to introduce something more about the network topology.   
+Network topology is the topological structure of a network, it refers to how nodes, devices in the network are physically or logically connected. There are a lot of different type of network topologies, including point-to-point, star, tree, mesh etc.   
+
+### 1.1.1. Point-to-Point
+Point-to-point topology is the simplest topology which has a dedicated link between two endpoints. Due to the direct link between these two devices, it can provide a fast and reliable connections, however, it can only be used for small area with only two devices involved.   
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/network-topology-point2point.png">  
+</div>  
+
+### 1.1.2. Star
+In star topology, every peripheral node is connected to a central node called a hub, and all traffic that traverses the network passes through the central hub, which acts as a signal repeater.   
+The advantage of the star topology is the simplicity of adding additional nodes, and each node in the network is independently connected to the central hub and will not affect others. On the flipside, since all of the peripheral are connected to the central hub and all peripheral communication must flow through the central hub, if the central hub goes down, the reset of the network cannot work anymore. And the network size will be limited by the capability of the central hub.   
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/network-topology-star.png">  
+</div>  
+
+### 1.1.3. Tree
+A tree topology can be viewed as a collection of star networks arranged in a hierarchy.   
+Unlike the star topology that each node is directly connected to the central hub, a tree topology has a parent-child hierarchy to how the nodes are connected. 
+The advantage of the tree network is that you can expand the network easily, and each branch of the network can be individually work.   
+
+Similar as the star topology, the whole network depends on the health the central hub, if the central hub goes down, the entire network cannot work. In the tree network, each branch of the tree network also depends on the health of the central device of the tree branch. And each two device in the network cannot communicate without the participation of the network or tree central device.   
```

Comment:
"the whole network depends on the health the central hub" --> "the whole network depends on the health of the central hub"